napari/layers/points/points.py

```diff
@@ -1516,7 +1516,8 @@ def _slice_data(
                 return slice_indices, scale
             else:
                 data = self.data[:, not_disp]
-                matches = np.all(data == indices[not_disp], axis=1)
+                distance = np.abs(data - indices[not_disp])
+                matches = np.all(distance < 1e-5, axis=1)
```

Comment:
I chose tolerance by random. I think that 10^(-5) ok, but I'm open.

Reply:
you could use https://numpy.org/doc/stable/reference/generated/numpy.allclose.html instead which has a default tolerance of 10^-8. Probably will be faster as i suspect they return as soon as they find something not close

Reply:
As i check allclose documentation thereis no axis argumen and allclose return single boolean operator. 

I think that you mean isclose, but isclose need to allocate arrays of size equal to points, because here right value is single row. 

This is why i use a manual way of comparision.

Reply:
ah ok, that's fair. not sure if @jni knows an alternative approach, otherwise your approach is fine